src/controllers/confirmation.controller.ts

```diff
@@ -120,8 +119,9 @@ const getOverdueFiling = ({isAccountsOverdue, isConfirmationStatementOverdue}):
     overdueFiling = "confirmation statement";
   } else if (isAccountsOverdue && isConfirmationStatementOverdue) {
     overdueFiling = "accounts and confirmation statement";
-  } else if (!isAccountsOverdue && !isConfirmationStatementOverdue) {
-    // TODO handle this output with appropriate render when story is created.
+  } else {
```

Comment:
The else logic was redundant since this was the only remaining scenario (plus it was preventing 100% coverage) so removed.